### DISCUSSION TITLE
Get only a specific field

### DISCUSSION THREAD

**Discussion by haf** [swellstores/community#101] [Help] [answered]
Can I filter the response, e.g.

    $ curl /products?only=tags -H 'accept:application/json'

?

This would enable me to avoid pagination and over-fetching.

**Comment by rodrigoddalmeida** [comment, marked as answer]
You can filter/allowlist which fields you want to return in the response object by using the `fields` param:

```js
// through the REST API

/products?fields[0]=tags

// or Swell.js

await swell.products.list({
  fields: ["tags"]
})
```
